prototypes/prototype.adoc

```diff
@@ -0,0 +1,44 @@
+---
+authors: Han Solo <han@example.com>, Frodo Baggins <frodo@example.com>
+state: predraft
+---
+= RFD <NUMBER> <Title>
+:author: Han Solo <han@example.com>; Frodo Baggins <frodo@example.com>
```

Comment:
Yup, thanks. I updated the new chatbot (joybot1999) already. I'll get the current jabber bot.